patternfly/patternfly-elements
# fix: pfe-autocomplete update aria-selected attribute

## <Pull request description>

Add feature to update aria-selected attribute to add the `aria-selected="true"` to the currently active list item when keying up and down through the droplist list items.


### Related issues

- n/a


### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Autocomplete Demo](https://deploy-preview-1713--patternfly-elements.netlify.app/elements/pfe-autocomplete/demo/) 

### Testing instructions



  1. Go to the [demo page](https://deploy-preview-1713--patternfly-elements.netlify.app/elements/pfe-autocomplete/demo/)
  2. Click in side of the autocomplete input
  3. Inspect the page and find the `.droplist ul li` items
  4. Starting type the letter `r` in the input
  5. Press the `arrow` down key and double check that the `aria-selected="true"` is added and removed as you key through the list
  6. The attribute should exist on the currently active list item only

#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
